### ISSUE TITLE
Problem with refreshing Android

### ISSUE THREAD

**Issue by zuzannaczaja** [dotnet/maui#8406]
### Description

I have problem with refreshing UI on Android platform after changing IsVisible variable.
The buttons disappeared, but when I clicked in entry and the keyboard appeared, the buttons appeared too. 

I created repository with project.

For example, on Windows platform I don't have this issue.

https://github.com/zuzannaczaja/BugRefreshUIAndroidMAUI

### Steps to Reproduce

1. Download project.
https://github.com/zuzannaczaja/BugRefreshUIAndroidMAUI
2. Open project and start on Android emulator.
3. Click on button.
4. Click on entry.

### Version with bug

Unknown/Other (please specify)

### Last version that worked well

Unknown/Other

### Affected platforms

Android

### Affected platform versions

Android 12, Android 11

**Comment by NonameMissingNo** [comment]
@zuzannaczaja Hey there, the repo that you've linked in isn't open to the public

**Comment by zuzannaczaja** [comment]
I set project as public.

**Comment by NonameMissingNo** [comment]
@kristinx0211 Hey Kristin, could you check on the Visual Studio version of Maui, I believe that the main branch is working.
![image](https://user-images.githubusercontent.com/40233496/176420387-6ce21597-4862-455f-ac93-43459286ec3b.png)

**Comment by zuzannaczaja** [comment]
> @kristinx0211 Hey Kristin, could you check on the Visual Studio version of Maui, I believe that the main branch is working. ![image](https://user-images.githubusercontent.com/40233496/176420387-6ce21597-4862-455f-ac93-43459286ec3b.png)

My video with bug

https://user-images.githubusercontent.com/56079308/176422657-8ee67046-7f16-48f6-9184-fb74db18b9ed.mp4

**Comment by NonameMissingNo** [comment]
Yup, that is no longer a problem on the main branch. ( net6.0 which can be downloaded from https://github.com/dotnet/maui/blob/main/.github/DEVELOPMENT.md )
You can either switch to that, or wait until the next service release is out.

**Comment by kristinx0211** [comment]
I can repro this issue on android with vs 17.3.0 Preview 3.0 [32629.436.main]. not repro on windows and mac.
https://user-images.githubusercontent.com/99457000/176645147-fc003a4a-2f57-4297-a4b0-29d8eb5c8b21.mp4

**Comment by zuzannaczaja** [comment]
After update (17.3.0 Preview 3.0) this bug is fixed
